src/scripts/main.js

```diff
@@ -1,7 +1,35 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector("body")
```

Comment:
It's recommended to cache frequently used selectors to improve performance. You're selecting the `body` every time the `pushNotification` function is called. Consider moving this line outside of the function.